modules/widgets/twitter-timeline.php

```diff
@@ -72,6 +72,13 @@ public function admin_scripts( $hook ) {
 					'modules/widgets/twitter-timeline-admin.js'
 				)
 			);
+
+			wp_enqueue_style(
+				'twitter-timeline-admin',
+				plugins_url( 'twitter-timeline.css', __FILE__ ),
```

Comment:
I wonder if we should use `admin` in the filename as well, to avoid any confusion.

Reply:
Fixed in 709a38758e3d9d9ee7bf15d22f3b7b7041737e57